### PR TITLE
Update `chain.schema.json` to `v15.2.0`

### PR DESCRIPTION
**Pull request by niccoloraspa**
## What is the purpose of the change

Update chain-registry to latest minor release `v15.2.0`

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [x] N/A

### CODE REVIEW

**Review by ValarDragon — APPROVED** [review]

File: chain.schema.json [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
+      "v15.2.0"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v15.1.0/osmosisd-15.1.0-linux-amd64?checksum=sha256:f16f901dfd47426caced8436c507e2950a4e2452b6cc6efc7845ab3b43a5c5d5",
-      "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v15.1.0/osmosisd-15.1.0-linux-arm64?checksum=sha256:08b20f0753b6b0bafa3a6ca7e4bb6807baf72754dfb69d6db7b1fb2b4b55ffc0"
+      "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v15.2.0/osmosisd-15.2.0-linux-amd64?checksum=sha256:3aab2f2668cb5a713d5770e46a777ef01c433753378702d9ae941aa2d1ee5618",
+      "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v15.2.0/osmosisd-15.2.0-linux-arm64?checksum=sha256:e158d30707a0ea51482237f99676223e81ce5a353966a5c83791d2662a930f35"
     },
-    "cosmos_sdk_version": "0.45",
+    "cosmos_sdk_version": "0.46.10",
```
Comment (ValarDragon) [inline chain.schema.json:16]:
```suggestion
    "cosmos_sdk_version": "0.45",
```

**Comment by niccoloraspa** [comment]
Thanks for the fix! I missed the redirect

**Comment by niccoloraspa** [comment]
I asked Jeremy to review and then I think we can merge. I have pinged him

**Comment by ValarDragon** [comment]
right now part of the problem is one CI job won't run, not sure why it didn't trigger. I suspect GH error

**Review by JeremyParish69 — APPROVED** [review]
LGTM

File: chain.schema.json [hunk 2]
```diff
@@ -83,27 +85,29 @@
...
         "name": "v15",
-        "tag" : "v15.1.0",
+        "tag" : "v15.2.0",
         "height": 8732500,
-        "recommended_version": "v15.1.0",
+        "recommended_version": "v15.2.0",
         "compatible_versions": [
+          "v15.2.0",
+          "v15.1.2",
           "v15.1.0",
```
Comment (ValarDragon) [inline chain.schema.json:94]:
```suggestion
```

Deleting this to try re-running CI

**Comment by niccoloraspa** [comment]
Re-ran CI. All passed.

**Pull request merged**